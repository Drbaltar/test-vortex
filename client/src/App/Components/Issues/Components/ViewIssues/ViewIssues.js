import React, { Component } from 'react';

export default class ViewIssues extends Component {
    render() {
        return <div></div>;
    }
}