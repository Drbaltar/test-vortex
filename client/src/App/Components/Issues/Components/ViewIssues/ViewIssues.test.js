import React from 'react';
import { shallow } from 'enzyme';
import ViewIssues from './ViewIssues';

describe('ViewIssues', () => {
    const wrapper = shallow(<ViewIssues />);

    it('renders correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});