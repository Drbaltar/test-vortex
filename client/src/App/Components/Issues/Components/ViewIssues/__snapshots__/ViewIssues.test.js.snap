// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewIssues renders correctly 1`] = `<div />`;
